certbot-dovecot/certbot_dovecot/tests/dovecotparser_test.py

```diff
@@ -0,0 +1,158 @@
+"""Tests for certbot_dns_cloudxns.dns_cloudxns."""
+
+import os
+import unittest
+
+import mock
+
+from certbot_dovecot import dovecotparser
+from certbot_dovecot.dovecotparser import DovecotParser
+
+
+class DovecotParserTest(unittest.TestCase):
+    def setUp(self):
+        self.parser_instance = DovecotParser()
+
+    def test_key_value(self):
+        string = "x = 1 a b c,d ,e, f\n"
+        parsed = self.parser_instance.parse_string(string).asList()
+
+        result = [list(string[0:-1])]
+        self.assertEqual(parsed, result)
```

Comment:
One thing we should be careful about is preserving whitespace, so we don't accidentally strip things from existing config files. For instance, here, `x = y` should parse differently than `x = y\n`, and `x = y` should parse differently from `x=y`.

From a first glance, most cases should be ok. I'd recommend adding more tests to make this behavior explicit.